### ISSUE TITLE
az network bastion rdp error following docs

### ISSUE THREAD

**Issue by WilliamDAssafMSFT** [Azure/azure-cli#23228]
## Describe the bug

**Command Name**
`az network bastion rdp`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)
Traceback (most recent call last):
  File "/opt/az/lib/python3.10/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/custom.py", line 8457, in rdp_bastion_host
    from ._process_helper import launch_and_wait
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/_process_helper.py", line 9, in <module>
    from ctypes import WinDLL, c_int, c_size_t, Structure, WinError, sizeof, pointer
ImportError: cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az network bastion rdp --name {} --resource-group {} --target-resource-id {}`

As explained here: https://docs.microsoft.com/en-us/azure/bastion/vm-upload-download-native#rdp the commands resulted in the above error

## Expected Behavior

Open RDP native client

## Environment Summary
```
Linux-5.4.0-1086-azure-x86_64-with-glibc2.28 (Cloud Shell), Common Base Linux Delridge (quinault)
Python 3.10.5
Installer: DEB

azure-cli 2.38.0

Extensions:
ai-examples 0.2.5
ssh 1.1.2

Dependencies:
msal 1.18.0b1
azure-mgmt-resource 21.1.0b1
```
## Additional Context

From Azure Cloud Shell in Windows 11 Terminal

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@WilliamDAssafMSFT Apologies for the late reply. Thanks for reaching out to us and reporting this issue. Could you please confirm if you are running this from a cloudshell ? If Yes, it wont be able to open the RDP on your local machine. For this reason, we recommend using powershell with az cli (on **_Windows_** machine). 

Also, note that RDP command works best on Windows, for other OS’ (**like Linux**) you can do the [tunnel command](https://docs.microsoft.com/en-us/azure/bastion/connect-native-client-windows#connect-tunnel) and then use the client of your choice.

>Connect to a VM using the [az network bastion tunnel command](https://docs.microsoft.com/en-us/azure/bastion/connect-native-client-windows#connect-tunnel). This lets you do the following:

>-  Use native clients on non-Windows local computers (example: **a Linux PC**).
>-  Use the native client of your choice. (This includes the Windows native client.)
>-  Connect using SSH or RDP.
>-  Set up concurrent VM sessions with Bastion.
>- [Upload files](https://docs.microsoft.com/en-us/azure/bastion/vm-upload-download-native#tunnel-command) to your target VM from your local computer. File download from the target VM to the local client is currently not supported for this command.

>Currently, this feature has the following limitation:
Signing in using an SSH private key stored in Azure Key Vault isn’t supported with this feature. Before signing in to your Linux VM using an SSH key pair, download your private key to a file on your local machine.

Hope this helps.

**Comment by navba-MSFT** [comment]
@WilliamDAssafMSFT I wanted to do quick follow-up to check if you had a chance to look at my above comment. Please let us know if you had a chance to follow the above suggestion. Awaiting your reply.

**Comment by WilliamDAssafMSFT** [comment]
I was using the Azure Cloud Shell within Windows Terminal locally on my Win11 device.

**Comment by navba-MSFT** [comment]
@WilliamDAssafMSFT Thanks for getting back. Could you please test this using PS with Az CLI ? That should help. Awaiting your reply.

**Comment by navba-MSFT** [comment]
@WilliamDAssafMSFT The action is currently pending on you to test` az network bastion rdp` CLI command using PS with Az CLI and update us the status. We will now proceed with closure of this GitHub issue. If you need any further assistance on this issue in future, please feel free to reopen this thread. We would be happy to help.

**Comment by WilliamDAssafMSFT** [comment]
@Navba-MSFT I get the same error in the Azure CLI in the Azure portal browser.
One thing I am quite unclear about in both the docs and the interaction is when I should be doing something in my local computer's Terminal or PowerShell session or in the Azure portal's built-in browser window. The doc says "Windows native clients" so I was using Windows Terminal.
![image](https://user-images.githubusercontent.com/74387232/182474562-1bc91839-c16c-4480-95f5-742f30cad4e2.png)

```
PS /home/william> az network bastion rdp --name "20220717_sql2022-vnet-bastion" --resource-group "20220717_sql2022" --target-resource-id "/subscriptions/d8392f63-30b0-4d4f-9ebb-90af66a30612/resourceGroups/20220717_sql2022/providers/Microsoft.Compute/virtualMachines/20220717-sql2022-ctp"                                    
Command group 'network bastion' is in preview and under development. Reference and support levels: https://aka.ms/CLI_refstatus
The command failed with an unexpected error. Here is the traceback:
cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)
Traceback (most recent call last):
  File "/opt/az/lib/python3.10/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/custom.py", line 8457, in rdp_bastion_host
    from ._process_helper import launch_and_wait
  File "/opt/az/lib/python3.10/site-packages/azure/cli/command_modules/network/_process_helper.py", line 9, in <module>
    from ctypes import WinDLL, c_int, c_size_t, Structure, WinError, sizeof, pointer
ImportError: cannot import name 'WinDLL' from 'ctypes' (/opt/az/lib/python3.10/ctypes/__init__.py)
To open an issue, please run: 'az feedback'
PS /home/william> 
```

**Comment by navba-MSFT** [comment]
@WilliamDAssafMSFT Thanks for getting back. Like I mentioned earlier, You will see this issue if you try to run the command from the portal Azure cloud shell. 

Please run the command from your local machine ( PS ). 
Don't run this command from the Azure portal browser.

Tagging another similar issue here. https://github.com/Azure/azure-cli/issues/23143 if that helps.

**Comment by gunsto** [comment]
We had this issue too. But it turned out that the name of the target server was wrong. It would be better to get a clear error message then.

**Comment by AllenOnAzure** [comment]
I ran this perfectly on my Windows 10 laptop. The same ps1 fails on Windows 11.

**Comment by jiasli** [comment]
Duplicate of #23143
